specification/search/resource-manager/Microsoft.Search/stable/2020-08-01/search.json

```diff
@@ -1671,15 +1632,63 @@
           "externalDocs": {
             "url": "https://azure.microsoft.com/documentation/articles/search-sku-tier/"
           }
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
         }
       },
       "allOf": [
         {
-          "$ref": "#/definitions/Resource"
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
```

Comment:
Why is this `TrackedResource` vs `Resource` ?

Reply:
ahh, is it because of `Location`?

Please confirm

Reply:
Our version of Resource was essentially comprised of the ARM's common TrackedResource + Identity.

Reply:
`Location` should be a required field, which is why it was changed to a `TrackedResource`. If a customer didn't provide a `Location`, we wouldn't know where to provision their service.

Reply:
ARM's common `Resource` would have been missing Location and Tags.